lib/modules/schedule/domain/bloc/schedule_bloc.dart

```diff
@@ -3,34 +3,45 @@ import 'dart:async';
 import 'package:bloc/bloc.dart';
 import 'package:freezed_annotation/freezed_annotation.dart';
 import 'package:injectable/injectable.dart';
+import 'package:intl/intl.dart';
+import 'package:ui_collection/core/mocked_entities/schedule_mocked_entities/schedule_mocked_entities.dart';
+
+import '../entities/schedule_item_entity.dart';
 
 part 'schedule_bloc.freezed.dart';
 part 'schedule_event.dart';
 part 'schedule_state.dart';
 
 @injectable
 class ScheduleBloc extends Bloc<ScheduleEvent, ScheduleState> {
-  ScheduleBloc() : super(const _Initial()) {
+  ScheduleBloc() : super(_Initial()) {
     on<ScheduleEvent>(
       (event, emit) => event.when(
-        increment: (value) => _handleIncrement(
-          emit,
-          value,
-        ),
+        load: (DateTime? currDate) {
+          return _handleLoad(emit, currDate);
+        },
       ),
     );
   }
 
-  FutureOr<void> _handleIncrement(
-    Emitter<ScheduleState> emit,
-    int value,
-  ) {
-    emit(const ScheduleState.initial());
+  Future<FutureOr<void>> _handleLoad(
+      Emitter<ScheduleState> emit, DateTime? currDate) async {
+    emit(const _Loading());
+    await Future.delayed(const Duration(milliseconds: 1000));
+
+    DateTime thisDate = currDate ?? DateTime.now();
+
+    List<ScheduleItemEntity> schedules =
+        ScheduleMockedEntities.getItems(thisDate);
 
-    value += 1;
+    String monthYear() {
```

Comment:
Function / Method Rule #5, Always have one purpose per function. 

Move this function outside